dashboard/src/index.html

```diff
@@ -8,12 +8,15 @@
 
   <link href="img/favicon.ico" rel="shortcut icon" type="image/vnd.microsoft.icon" />
   <!-- STYLES -->
-  <link rel="stylesheet" type="text/css" href="css/index.css">
   <!-- build:css lib/css/main.min.css -->
   <link rel="stylesheet" type="text/css" href="components/bootstrap/dist/css/bootstrap.min.css">
   <link rel="stylesheet" type="text/css" href="components/font-awesome/css/font-awesome.min.css">
   <link rel="stylesheet" type="text/css" href="components/rdash-ui/dist/css/rdash.min.css">
   <!-- endbuild -->
+
```

Comment:
@cossou Of course, I saw know how gulp compiles from less and minifies and all the rest. What I'm looking out for is say... I check the login page and I try 'Inspect Element' in my browser, I can then see which CSS files are loading which parameters for an element I'm viewing. Now having the CSS in one file means that no matter what the code, it will all show from the one file.

That will make it hard for us to find the code as it could be in one of the 3 files we get externally and minify or it could be in our custom css file but we wouldn't be able to quickly look and identify it.

Since we keep going back and forth, here's two options. Let me know which you choose and I can help out on it further:

A) I help setup a gulp task that minifies the current index.css file (our custom css) to a minified index file. Then search in the HTML for places to replace index.css with index.mini.css.

B) If you extremely want to, I simply move the CSS code up to be minified with the main.min.css goods. I don't recommend doing this though because of the reasons I mentioned above.